core/templates/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -90,14 +90,18 @@ export class ExplorationEngineService {
      private statsReportingService: StatsReportingService,
      private urlService: UrlService
    ) {
+     this.setExplorationProperties();
+   }
+
+   setExplorationProperties(): void {
      let pathnameArray = this.urlService.getPathname().split('/');
      let explorationContext = false;
 
      for (let i = 0; i < pathnameArray.length; i++) {
        if (pathnameArray[i] === 'explore' ||
-            pathnameArray[i] === 'create' ||
-            pathnameArray[i] === 'skill_editor' ||
-            pathnameArray[i] === 'embed') {
+             pathnameArray[i] === 'create' ||
```

Comment:
Spacing in this file is a bit of an anti-pattern.
All lines are spaced in odd numbers (3, 5, ...) instead of even.
Should we catch these in lint checks @seanlip @Hudda ?

Reply:
Yeah, I think this should be caught in lint checks. This is more a question for @Hudda and @DubeySandeep though.

Reply:
@seanlip @Radesh-kumar This issue is already tracked in #8423